.gitignore

```diff
@@ -34,6 +34,10 @@
 # Build folders
 build/
 cmake-build-debug/
+Aquarius/Testing/
+CMakeFiles/
+Aquarius/Aquarius/
+Aquarius/Sandbox/Makefile
```

Comment:
Are we sure this is necessary? Did you ever look into this?

Reply:
Removing these, I reconfigured my cmake.